tests/shared/robot.spec.js

```diff
@@ -131,16 +131,16 @@ describe('Shared: Robot', function() {
         data: {
           message: 'foo',
           code: 'MYERROR',
-          stacktrace: error.stack
+          stack: error.stack
         }
       });
     });
 
     it('should handle a string error', function() {
```

Comment:
> Fair enough. But if robot.printError accepts a string "just in case", then you ought to do the same for utils.printError because it appears just a couple of lines below robot.printError in lib/cli/etcher.js (with the same error variable being passed to both).

`utils.printError` does handle strings already, since it re-uses the same `lib/shared/errors.js` code than `robot.printError`.